markdown/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -27,15 +27,19 @@ This page describes all of the Handlebars helpers supported on the Stencil frame
 
 For background information on using Handlebars helpers, please see the [official Handlebars documentation](http://handlebarsjs.com).
 
+---
+
 <a href='#handlebars-helpers-reference_array' aria-hidden='true' class='block-anchor'  id='handlebars-helpers-reference_array'><i aria-hidden='true' class='linkify icon'></i></a>
 
-# <a name="array"></a> Array Helpers
+## <a name="array"></a> Array Helpers

```

Comment:
You have to put the anchor tag and the title on separate lines or else Stoplight has problems using them. They work inconsistently. 

```
<a name ="array"></a>

## Array Helpers
```

https://community.stoplight.io/t/markdown-anchors-not-working/746

Reply:
Is there a ticket for this?